src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/ProcessHmcMessageService.java

```diff
@@ -60,6 +61,10 @@ public void processEventMessage(HmcMessage hmcMessage)
 
         SscsCaseData caseData = ccdCaseService.getCaseDetails(caseId).getData();
 
+        if (LISTED == hmcStatus) {
+            caseData.setDwpState(HEARING_DATE_ISSUED.getId());
```

Comment:
Do you need to set hmctsDwpState for hmcts cases?

Reply:
As far as I've understood from the ticket and Hurman's handover, this is what was required. @dattipoeHMCTS / @ss-evoco might be able to clarify or shed some light if this is in fact the right direction for this change.

Reply:
@nickhill111 can you confirm if this is needed?